### DISCUSSION TITLE
Streaming

### DISCUSSION THREAD

**Discussion by LeonWehrhahn** [gavanitrate/faunadb-http-dart#21] [Ideas]
Are you planning on adding [streaming](https://docs.fauna.com/fauna/current/drivers/streaming) to this package?

**Comment by gavanitrate** [comment]
I don't have any concrete plans for that yet. I will probably look into potential implementation in the next week or two. If any one has advice, it would be great to hear it.

**Comment by AndreBaltazar8** [comment]
I've implemented a basic version of streaming here: https://github.com/BSTEVRrepo/faunadb-http-dart

It's basically just getting the event types and emitting those right now. The API is also a bit strange but I tried to make it good for Dart, and based on the JS one. It works enough for my purposes that require just knowing when there is a new version of the document.
